app/javascript/react/components/App.js

```diff
@@ -11,6 +12,7 @@ export const App = (props) => {
         <Route exact path="/" component={NetflixIndexContainer}/>
         <Route exact path="/netflix_shows" component={NetflixIndexContainer} />
         <Route exact path="/netflix_shows/:id" component={NetflixShowContainer} />
+        <Route exact path="/netflix_shows/:id/add_review" component={NetflixFormContainer} />
```

Comment:
unfortunately, your review form should be ON the show page. Remove line 15 entirely, and instead add your form container as a child on the show page. further, the component should be renamed `ReviewFormContainer`